src/common/styles/colors.scss

```diff
@@ -57,7 +57,7 @@
     --neutral-alpha-80: rgba(0, 0, 0, 0.8);
     --neutral-alpha-90: rgba(0, 0, 0, 0.9);
     // Outcome colors
-    --positive-outcome: #107c10;
+    --positive-outcome: #228722;
```

Comment:
??? is this intended? did you check with Chelsea about this?

Reply:
This is the green color on the latest Figma for the new report.